### PR TITLE
Remove 3.8 (EOL)

### PR DESCRIPTION
**Pull request by tianon**
https://github.com/docker-library/rabbitmq/commit/6e226fe8e99702c8726d5e7d5c5864e69548048d#commitcomment-73250084 :partying_face:

### CODE REVIEW

**Comment by michaelklishin** [comment]
I'd wait until 3.8.30 ships and we put up one more tag.

**Comment by michaelklishin** [comment]
Actually, maybe we should wait until July or so as we expect a couple more `3.8.x` releases.

**Comment by tianon** [comment]
:joy: fair -- I've converted this to a draft for now :+1:

**Comment by michaelklishin** [comment]
@tianon one month and two patch release rounds later, I think we can pull the trigger on this one :)

File: Dockerfile-alpine.template [hunk 1]
```diff
@@ -12,7 +12,7 @@ RUN apk add --no-cache \
 # Default to a PGP keyserver that pgp-happy-eyeballs recognizes, but allow for substitutions locally
 ARG PGP_KEYSERVER=keyserver.ubuntu.com
 # If you are building this image locally and are getting `gpg: keyserver receive failed: No data` errors,
-# run the build with a different PGP_KEYSERVER, e.g. docker build --tag rabbitmq:3.8 --build-arg PGP_KEYSERVER=pgpkeys.eu 3.8/ubuntu
+# run the build with a different PGP_KEYSERVER, e.g. docker build --tag rabbitmq:3.10 --build-arg PGP_KEYSERVER=pgpkeys.eu 3.10/ubuntu
```
Comment (tianon) [inline Dockerfile-alpine.template:15]: I guess I could replace this with `{{ env.version }}`...  hmm

@yosifkit thoughts?  worthwhile?  (I'm leaning towards probably overkill - maybe next time :joy:)

**Comment by tianon** [comment]
Oops: https://github.com/rabbitmq/rabbitmq-server/releases/tag/v3.8.35

We should probably revert this to get in 3.8.35, right?

**Comment by michaelklishin** [comment]
@tianon nope, 3.8 is now entirely out of support, so I wouldn't worry about it. It would take a serious vulnerability for us to produce a new release and request a temporary exception.

**Pull request merged**